db/migrations/20170629163043_add-popularity.js

```diff
@@ -0,0 +1,22 @@
+
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.alterTable('folders', (table) => {
+      table.dropUnique('folder_name')
```

Comment:
I'm bummed this didn't work the way we wanted to... though I wonder if we could refactor to keep it and do a check if it exists then grab the primary key and set the new url OR create the folder like we have it.

Good job finding the dropUnique method! Knex is cool.